webapp/home/labs/genome/annotation.yml

```diff
@@ -135,6 +135,136 @@ tabs:
         view_link: ''
         view_tip: ''
         button_tip: Run in Galaxy AU
+        
+      subsections:
+       - id: TSI transcripts
+          title: Transcript alignment 
+          content:
+            - title_html: About these workflows
+              description_html: >
+                <p>
+                  This <a href="https://australianbiocommons.github.io/how-to-guides/genome_annotation/Transcriptome" target="_blank"> How-to-Guide </a> will describe the steps required to align transcript data to your genome on the Galaxy Australia platform, using multiple workflows. The outputs from these workflows can then be used as inputs into the next annotation workflow using FgenesH++.
+                </p>
+            - title_html: Repeat masking
+              description_html: >
+                <p>
+                  Mask repeats in the genome.
+                </p>
+              inputs:
+                - datatypes:
+                    - fasta
+                  label: Assembled genome <em>genome.fasta</em>
```

Comment:
If `genome.fasta` is supposed to be a file then this should probably use <code> tags (and other instances below)